### ISSUE TITLE
Auto dismantle doesn't work if Python version mismatch

### ISSUE THREAD

**Issue by gilv** [lithops-cloud/lithops#397]
When python version mismatch, auto dismantle doesn't work for Gen2

In my case I got
```
Exception: The indicated runtime 'ibmfunctions/lithops-metabolomics-cf-3.8.5-2.2.0:1.0.0' is running Python 3.8 and it is not compatible with the local Python version 3.7
```

VM remained started forever.

I assume to resolve this, in case of such exception we need to shut it down from the client and not assume auto dismantle will do it

@kpavel @JosepSampe

**Comment by JosepSampe** [comment]
@gilv The proxy service is started as a unix service when the VM is started, at the same time, Gen1 credentials are always stored in the VM, so now the VM is stopped even if you don't submit a Job the job fails, or any other issue.

Note that `hard dismantle timeout` is 1 hour, not 5 minutes. Did you ensure that it was not stopped after 1 hour?

Maybe the error could come from another part, can you post logs of the VM proxy stored at /tmp/lithops/proxy.logs?

**Comment by JosepSampe** [comment]
I improved StandaloneExecutor logic in #400. It is ready to test.

All the tests I made was with Ubuntu 20.04, since now it is already possible to create VM instances with Ubuntu 20.04 in IBM VPC. 
It is worth to note that Ubuntu 20.04 VMs take ~35 seconds to start, in contrast of Ubuntu 18.04 VMs that take ~70 seconds. So I strongly recommend to create Ubuntu 20.04 VMs to substantially reduce VM startup time.
